azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/participantlist/ParticipantListView.kt

```diff
@@ -324,11 +324,13 @@ internal class ParticipantListView(
             participantViewData,
             isOnHold,
             onClickAction = {
-                if (status == ParticipantStatus.IN_LOBBY) {
-                    showAdmitDialog(displayName, userIdentifier)
-                } else if (accessibilityManager.isEnabled) {
-                    participantListDrawer.dismiss()
-                }
+                participantListDrawer.dismiss()
+                // TODO: uncomment when admit all button is supported
+//                if (status == ParticipantStatus.IN_LOBBY) {
+//                    showAdmitDialog(displayName, userIdentifier)
+//                } else if (accessibilityManager.isEnabled) {
```

Comment:
It looks like you have commented out too much here. Are you sure you need to remove non-lobby case?